### PR TITLE
Add support for uint64, int64

### PR DESCRIPTION
**Pull request by IzabellaRaulin**
### Summary of changes:
- added uint64 and int64 support

### Testing done:
- manually run task with collector which exposes metrics with different types of data
- run unit tests

@intelsdi-x/plugin-maintainers

### CODE REVIEW

**Comment by andrzej-k** [comment]
Please update date in licence headers, other than that LGTM.

**Comment by IzabellaRaulin** [comment]
@andrzej-k,  licence headers have been updated

File: postgresql/postgresql.go [hunk 1]
```diff
@@ -237,14 +244,58 @@ func interfaceToString(face interface{}) (string, error) {
...
+			return ret, err
+		}
+		for i := 1; i < length; i++ {
+			ret += ", "
+			ret += strconv.FormatFloat(val[i], 'g', -1, 64)
+		}
+
 	default:
-		err = errors.New("unsupported type")
+		err = fmt.Errorf("Unsupported type %v (currently supported data type: string, []string, bool, int, []int, int64, uint64, []uint64, float64, []float64)", reflect.TypeOf(val))
```
Comment (IRCody) [inline postgresql/postgresql.go]: Would it be worth adding `uint` and `[]int64`  so that the support is consistent between int/unit?

Comment (IzabellaRaulin) [inline postgresql/postgresql.go]: Thanks @IRCody, I added that to make this pull request ready to be merged.

**Comment by andrzej-k** [comment]
LGTM

**Pull request merged**